Server/BackEndMonitoringServer/src/CContainerOfProcesses.cpp

```diff
@@ -0,0 +1,107 @@
+#include "stdafx.h"
+
+#include <windows.h>
+#include <psapi.h>
+#include <algorithm>
+
+#include "CContainerOfProcesses.h"
+
+CContainerOfProcesses::CContainerOfProcesses(unsigned m_max_process_count,
+											std::chrono::duration<int> 
+											pause_duration,
+											std::string path_to_file,
+											EMemoryCountType count_type) :
+	m_max_process_count(m_max_process_count),
+	�HardwareStatusSpecification(pause_duration, path_to_file, count_type)
+{
```

Comment:
Do not do hard functionality in the constructor

Reply:
Fixed, moved to hard initialization to Initialize method, added bool field to check initialization, all getters return bool now.
[commit](https://github.com/ITA-Dnipro/BackEndMonitoring/commit/28775d4e4e211a6985cecfb05174a510800a77d4)